agent/src/agent/cli/backup.py

```diff
@@ -49,22 +51,21 @@ def _get_pipelines():
     return existing, not_existing
 
 
-def _update_status(pipeline_: pipeline.Pipeline):
-    if _should_change_status(pipeline_):
-        # we care only if it's running or stopped
-        if pipeline.manager.is_running(pipeline_.name):
-            # then stop it
-            pipeline.manager.stop(pipeline_)
-        else:
-            # then start it
-            if not _can_start(pipeline_):
-                pipeline.manager.stop(pipeline_)
-            pipeline.manager.start(pipeline_)
-
-
-def _should_change_status(pipeline_: pipeline.Pipeline) -> bool:
-    return pipeline_.status != pipeline.manager.get_pipeline_status(pipeline_.name)
+def _update_status(pipeline_: Pipeline):
+    for function in _change_status_functions(pipeline_):
+        function(pipeline_)
```

Comment:
why do you need this, why don't just call the functions at once?

Reply:
dunno :)